### ISSUE TITLE
SecondaryPantheon for Beliefs

### ISSUE THREAD

**Issue by JFDaly** [LoneGazebo/Community-Patch-DLL#3728]
Could you please add the SecondaryPantheon tag and functionality to the Beliefs table - this is the one that allows cities to benefit from the second most popular pantheon bonus, which you once added for the Buildings table. This is intended for a Founder belief. Thanks.

**Comment by LoneGazebo** [comment]
Unfortunately it’s not that easy - it would require over 100 entries and changes to the dll to adjust for the data. The policy one barely worked before, and barely works now...

**Comment by JFDaly** [comment]
Otru. Well, originally, I wanted the player to be able to pick an actual second Pantheon, but discovered that Game.EnhancePantheon() essentially enhances the Religion, which is not what I intended. So I opted for this effect thinking it would be easier to implement. Maybe the original idea would be easier? To make it so that Game.EnhancePantheon() doesn't enhance the Religion. 

If not, then no problem, I'll do something else.
